packages/daemon/src/worker.js

```diff
@@ -59,6 +59,17 @@ export const makeWorkerFacet = (getDaemonBootstrap, cancel) => {
       const namespace = await import(path);
       return namespace.main0(powerBox);
     },
+
+    importBundle0: async readable => {
+      const bundleText = await E(readable).text();
+      const bundle = JSON.parse(bundleText);
+
+      const { importBundle } = await import('@endo/import-bundle');
```

Comment:
Why the dynamic import?

Reply:
Possibly a premature optimization, but it seemed wasteful to bring in the import-bundle machinery for workers that don’t use it. That is, workers that just evaluate confined programs, for now.

Reply:
ah. makes sense.

perhaps worth a comment.

Reply:
Sure thing. A commit will appear in my next push.